### ISSUE TITLE
check: create Jenkinsfile for running an experiment

### ISSUE THREAD

**Issue by ivotron** [systemslab/popper#69]

**Comment by ivotron** [comment]
@shamedian I've pushed `wip-69` (see https://github.com/systemslab/popper/compare/master...wip-69), which adds a very simple `Jenkinsfile`. I've also pushed an [image to docker hub](https://hub.docker.com/r/ivotron/jenkins-popperci/) that you can use to test your code against. To launch this image:

```bash
docker run --name=jenkins \
  -u root \
  -e JENKINS_HOME=/var/lib/jenkins \
  -p 8080:8080 \
  -p 50000:50000 \
  -v /usr/bin/docker:/usr/bin/docker \
  -v /var/run/docker.sock:/var/run/docker.sock \
  -v /tmp:/tmp \
  ivotron/jenkins-popperci:2.32.3
```

This image has a test job with `REPO_URL` and `EXPERIMENT` parameters. This job should be able to run Popperized single-node experiments that depend only on docker/bash like [this one](https://github.com/systemslab/popper/tree/master/templates/experiments/blis). To test, I've created a repo so you can provide `REPO_URL=https://github.com/ivotron/sample-paper` and `EXPERIMENT=blis` and it should work on. Let me know if it works

**Comment by ivotron** [comment]
closed by #74

**Comment by ivotron** [comment]
forgot to mention that the plugins in the image are the following:

  * Pipeline
  * Credentials
  * Credentials Binding
  * Git
  * SSH agent
  * User env vars
